### PR TITLE
GitIssue-881: remove pip installtion through easy_install and add lnet validator

### PR DESCRIPTION
**Pull request by sachitanands**
Remove easy_install pip command from opensource init.sh script for dev vm.

Signed-off-by: Sachitanand Shelake <sachitanand.shelake@seagate.com>

### CODE REVIEW

**Comment by vimalraghwani20** [comment]
image
@sachitanands  i see more three places we have easy_install. 
check they are relevant or not.

**Review by vimalraghwani20 — APPROVED** [review]

**Comment by sachitanands** [comment]
> image
> @sachitanands i see more three places we have easy_install.
> check they are relevant or not.
I have handled script references in this PR itself.
In case of Quickstart guide , we need to verify these steps by setting up s3 client .. lets chekc with Patrick once if he can confirm if we require this step or not.

https://github.com/Seagate/cortx-s3server/blob/main/docs/CORTX-S3%20Server%20Quick%20Start%20Guide.md#14-test-your-build-using-s3-cli

**Review by nileshgovande — APPROVED** [review]

**Comment by nileshgovande** [comment]
Add lnet validator here as well @vimalraghwani20

File: scripts/env/common/configure_lnet.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  lctl list_nids | grep "@tcp" &> /dev/null
+  if [ $? -eq 0 ]; then
+    echo "Lnet is up and running"
+  else
+    echo "Lnet is down."
+    echo "configuring lnet"
+    echo "options lnet networks=tcp(eth0) config_on_load=1" > '/etc/modprobe.d/lnet.conf'
+    service lnet restart
+    # it has been observed that the lnet does not get restart in first go
+    # so for safer side restarting again and then verify
```
Comment (nileshgovande) [inline scripts/env/common/configure_lnet.sh:32]: wanna add a sleep in between ?

Comment (vimalraghwani20) [reply scripts/env/common/configure_lnet.sh:32]: @nileshgovande  we can. 
however, i tested above on my dev vm and without sleep it is working fine

Comment (vimalraghwani20) [reply scripts/env/common/configure_lnet.sh:32]: fixed

**Comment by vimalraghwani20** [comment]
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/1047/
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Mini-Provisioning/job/cortx-s3server/1060/

**Pull request merged**